### PR TITLE
Refactor stats

### PR DESCRIPTION
**Pull request by dometto**

### CODE REVIEW

**Comment by bartkamphorst** [comment]
Can you remove `spec/.DS_Store` from this PR? 😉

File: lib/commit.rb [hunk 1]
```diff
@@ -41,10 +41,7 @@ def stats
       df.set_repository(@jrepo)
       df.set_context(0)
       parent_commit = @jcommit.parent_count > 0 ? @jcommit.get_parents[0] : nil
-
-      rename_detector = RenameDetector.new(@jrepo)
-      rename_detector.addAll(df.scan(parent_commit, @jcommit))
-      entries = rename_detector.compute
+      entries = df.scan(parent_commit, @jcommit)
```
Comment (bartkamphorst) [inline lib/commit.rb]: Shouldn't we then set `setDetectRenames(true)`?

Comment (dometto) [reply lib/commit.rb]: I tested this and it's already true! The rename spec passes so all is fine :)

**Comment by dometto** [comment]
Oops, forgot to add the spec for renames in stats.

I've also changed the keys in the #stat result hash from `:old_path` to `:old_file` and `:new_path` to `:new_file`. I find this slightly less intuitive myself, but it's what rugged does, so it's good for conformity.

File: lib/commit.rb (hunk 1 above)
Comment (bartkamphorst) [reply lib/commit.rb]: OK.

Comment (dometto) [reply lib/commit.rb]: Strangely I didn't need to explicitly enable `setDetectRenames` locally, but the test on Travis was failing. So added it anyway.

**Comment by dometto** [comment]
@bartkamphorst any idea why Travis could be failing on Openjdk? Doesn't seem related to this PR

**Comment by dometto** [comment]
> Can you remove spec/.DS_Store from this PR? 😉
:) Done, and added to `.gitignore`

**Pull request merged**